meeting_minutes/211209_SARIF_TC_49.md

```diff
@@ -0,0 +1,171 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company          | Role(s)                 |
+| :--------- | :-------- | :--------------- | :---------------------- |
+| Aditya     | Sharad    | Microsoft        | Member                  |
+| Chad       | Bentz     | defi SOLUTINS    | Public Participant      |
+| David      | Keaton    | Individual       | Chair                   |
+| Eddy       | Nakamura  | Microsoft        | Voting member           |
+| Katrina    | O'Neil    | Micro Focus      | Voting member           |
+| Nathan     | Baird     | Microsoft        | Member                  |
+| Paul       | Anderson  | Grammatech, Inc. | Voting member           |
+| Stefan     | Hagen     | Individual       | Secretary, taking notes |
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/committees/download.php/69395/agenda_20211209.html
+
+  Agenda was approved
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2021-11-11 Meeting #48](https://www.oasis-open.org/committees/document.php?document_id=69391&wg_abbrev=sarif)
+
+  Minutes were approved
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION Michael: Reach out to contacts at Google and Uber on the Code Insights Protocol topic 
+  - ONGOING
+* ACTION Michael: Reach out to ParaSoft
+  - ONGOING
+* ACTION David: Provide draft of Wikipedia article for review on next meeting
+  - DONE
+* ACTION Michael: Will author and check in the roadmap / list of in- and out-of-scope features
+  - ONGOING
+ 
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+All voting members present besides Michael (who sent regrets).
+
+### 1.7.1 Prospective members attending their first meeting
+
+Aditya Sharad and Nathan Baird both will gain voting rights at the end of next meeting.
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair David)
+
+* Approved Teleconferences (Thursdays at 08:00 PT / 16:00 UTC for 1.5 hours)
+  ```
+  January 13
+  January 27
+  ```
+* Possible face-to-face meeting in 2022Q1
+
+# 3. Discussion
+
+## 3.1 Review recruitment effort to complete the technical committee
+
+* None
+
+## 3.2 Review status on finalizing SARIF 2.1 errata
+
+* Shall the TC approve the combination of these documents as Committee Specification Draft Errata?
```

Comment:
I am submitting the request to OASIS, and they have asked that the exact title appear in the minutes.  I suggest adding the following line before the web links.

Title:  Static Analysis Results Interchange Format (SARIF) Version 2.1.0 Errata CSD01

Reply:
Hope the added change does accomplish this.